From e1ae3e12a9c0edac64692d171dbed3b4011b0bfb Mon Sep 17 00:00:00 2001
From: Icenowy Zheng <icenowy@aosc.io>
Date: Wed, 17 Mar 2021 15:00:39 +0800
Subject: [PATCH 445/456] dt-bindings: vendor-prefix: add prefix for Voltafield

Voltafile Technology Corp. is a company that produces MEMS sensors.

Add a DT vendor prefix for it.

Signed-off-by: Icenowy Zheng <icenowy@aosc.io>
---
 Documentation/devicetree/bindings/vendor-prefixes.yaml | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/Documentation/devicetree/bindings/vendor-prefixes.yaml b/Documentation/devicetree/bindings/vendor-prefixes.yaml
index 294093d45a23..0e9c167f4425 100644
--- a/Documentation/devicetree/bindings/vendor-prefixes.yaml
+++ b/Documentation/devicetree/bindings/vendor-prefixes.yaml
@@ -1254,6 +1254,8 @@ patternProperties:
     description: Tronsmart
   "^truly,.*":
     description: Truly Semiconductors Limited
+  "^voltafield,.*":
+    description: Voltafield Technology Corp.
   "^visionox,.*":
     description: Visionox
   "^team-source-display,.*":
-- 
2.34.1

